pyproject.toml

```diff
@@ -25,6 +25,7 @@ sqlalchemy = "^2.0.31"
 langchain-community = "^0.2.12"
 neo4j = "^5.23.1"
 rank-bm25 = "^0.2.2"
+google-generativeai = "^0.8.1"
```

Comment:
Hey, we want to make keep this as optional dependency

Reply:
I wanted to learn how to pass ci without adding this, 

```
E   ModuleNotFoundError: No module named 'google.generativeai'
```

Reply:
@PranavPuranik You should add the library in `make` file under `install_all`